easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023b.eb

```diff
@@ -0,0 +1,210 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '2.15.1'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'pic': True}
+
+use_pip = True
+sanity_pip_check = True
+
+builddependencies = [
+    ('Bazel', '6.3.1'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.42.0'),
+    ('pybind11', '2.11.1'),
+    ('UnZip', '6.0'),
+    # Required to build some of the extensions
+    ('poetry', '1.6.1'),
+    # Protobuf disabled since 2.13.0 easyconfigs:
+    # Compiling with system protobuf don't seem to work, see:
+    # https://github.com/tensorflow/tensorflow/issues/61593
+    # ('protobuf', '24.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('h5py', '3.11.0'),
+    ('cURL', '8.3.0'),
+    ('dill', '0.3.8'),
+    ('double-conversion', '3.3.0'),
+    ('flatbuffers', '23.5.26'),
+    ('flatbuffers-python', '23.5.26'),
+    ('giflib', '5.2.1'),
+    ('hwloc', '2.9.2'),
+    ('ICU', '74.1'),
+    ('JsonCpp', '1.9.5'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('ml_dtypes', '0.4.0'),
+    ('NASM', '2.16.01'),
+    ('nsync', '1.29.2'),
+    ('SQLite', '3.43.1'),
+    ('patchelf', '0.18.0'),
+    ('libpng', '1.6.40'),
+    ('snappy', '1.1.10'),
+    ('zlib', '1.2.13'),
+    ('grpcio', '1.57.0'),
```

Comment:
Please change to the already merged 1.67.1 version of grpcio and drop the 1.57.0 one from the PR